osu.Game/Overlays/BeatmapSet/Scores/TopScoreUserSection.cs

```diff
@@ -28,7 +28,7 @@ public class TopScoreUserSection : CompositeDrawable
         private readonly SpriteText date;
         private readonly UpdateableFlag flag;
 
-        public TopScoreUserSection()
+        public TopScoreUserSection(int position)
```

Comment:
this can't be in the `ctor` is you leave the `score` as a property. it makes no sense, as you are able to update one but not the other.